metals/src/main/scala/scala/meta/internal/metals/Command.scala

```diff
@@ -1,20 +1,82 @@
 package scala.meta.internal.metals
 
+import scala.reflect.ClassTag
+import scala.util.matching.Regex
+
 import scala.meta.internal.metals.MetalsEnrichments._
 
 import com.google.gson.JsonPrimitive
 import org.eclipse.{lsp4j => l}
 
+trait BaseCommand {
+  def id: String
+  def title: String
+  def description: String
+  def arguments: String
```

Comment:
Plain `arguments` look strange to me, perhaps?
```suggestion
  def commandArguments: String
```

Reply:
It's arguments in the command trait/classes, so I think this is fine.